webpack/dev.config.js

```diff
@@ -1,69 +1,40 @@
 import webpack from 'webpack';
 import git from 'git-rev-sync';
-import { isArray } from 'lodash';
 import baseConfig from './base.config';
 import startKoa from './utils/start-koa';
 
-const LOCAL_IP = require('dev-ip')();
-const WEBPACK_PORT = process.env.PORT ? parseInt(process.env.PORT)+1 : 8081;
-const HOST = (isArray(LOCAL_IP) && LOCAL_IP[0]) || LOCAL_IP || 'localhost';
-const PUBLIC_PATH = "/assets/";
+// var BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
+// baseConfig.plugins.push(new BundleAnalyzerPlugin());
 
 export default {
-    server: {
-        port: WEBPACK_PORT,
-        options: {
-            publicPath: PUBLIC_PATH,
-            hot: true,
-            stats: {
-                assets: true,
-                colors: true,
-                version: false,
-                hash: false,
-                timings: true,
-                chunks: false,
-                chunkModules: false
-            }
-        }
+    ...baseConfig,
```

Comment:
take that, `Object.assign`!

I really love the object spread operator, it's the best thing in Javascript since `=>`